templates/home-assistant/0/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  home-assistant:
+    image: homeassistant/home-assistant:0.51.2
+    privileged: true

```

Comment:
I take a look at https://home-assistant.io/docs/installation/docker/, Is privileged mode mandatory??

Reply:
Well, yes and no.  If you are using Z-wave / Insteon / etc for automation those are devices that are plugged into the host machine.  So, IF you are using those (which most people are) then yes (that said there are work arounds, but they aren't for the faint of heart)

Reply:
If it isn't mandatory, what do you think to add a question in rancher-compose?? :)
```
-    privileged: true
+    privileged: ${privileged}
```